src/algebra/ordered_field.lean

```diff
@@ -655,4 +657,24 @@ by rw [abs_div, abs_one]
 lemma abs_inv (a : α) : abs a⁻¹ = (abs a)⁻¹ :=
 (abs_hom : monoid_with_zero_hom α α).map_inv' a
 
+-- TODO: add lemmas with `a⁻¹`.
+lemma one_div_strict_mono_decr_on : strict_mono_decr_on (λ x : α, 1 / x) (set.Ioi 0) :=
+λ x x1 y y1 xy, (one_div_lt_one_div (set.mem_Ioi.mp y1) (set.mem_Ioi.mp x1)).mpr xy
+
+lemma one_div_pow_le_one_div_le_pow (a1 : 1 ≤ a) {m n : ℕ} (mn : m ≤ n) :
+  1 / a ^ n ≤ 1 / a ^ m :=
+by refine (one_div_le_one_div _ _).mpr (pow_le_pow a1 mn);
+  exact pow_pos (lt_of_lt_of_le zero_lt_one a1) _
+
+lemma one_div_pow_lt_one_div_pow_lt (a1 : 1 < a) {m n : ℕ} (mn : m < n) :
+  1 / a ^ n < 1 / a ^ m :=
+by refine (one_div_lt_one_div _ _).mpr (pow_lt_pow a1 mn);
+  exact pow_pos (trans zero_lt_one a1) _
+
+lemma one_div_pow_mono (a1 : 1 ≤ a) : monotone (λ n : ℕ, order_dual.to_dual 1 / a ^ n) :=
+λ m n mn, one_div_pow_le_one_div_le_pow a1 mn
+
+lemma one_div_pow_strict_mono (a1 : 1 < a) : strict_mono (λ n : ℕ, order_dual.to_dual 1 / a ^ n) :=
+λ m n mn, one_div_pow_lt_one_div_pow_lt a1 mn
```

Comment:
Don't you mean
```suggestion
lemma one_div_pow_mono (a1 : 1 ≤ a) : monotone (λ n : ℕ, order_dual.to_dual (1 / a ^ n)) :=
λ m n mn, one_div_pow_le_one_div_le_pow a1 mn

lemma one_div_pow_strict_mono (a1 : 1 < a) : strict_mono (λ n : ℕ, order_dual.to_dual (1 / a ^ n)) :=
λ m n mn, one_div_pow_lt_one_div_pow_lt a1 mn
```

Reply:
I removed the parentheses, although they eventually mean the same thing, right?